test/benchee_test.exs

```diff
@@ -253,6 +253,40 @@ defmodule BencheeTest do
     end
   end
 
+  test ".run accepts atom keys for jobs" do
+    capture_io fn ->
```

Comment:
:wave: 

Yup that's what I'm using it for, imo test output shouldn't show any CLI output at all. I'm just not quite sure what you mean with as a feature in `ExUnit` - you mean like a tag or something that captures all IO output for a given test? Because `capture_io` already comes from `ExUnit` :)